src/test/java/JorgeCaceresTest/Facade/JsonBodyTest.java

```diff
@@ -0,0 +1,32 @@
+package JorgeCaceresTest.Facade;
+
+import JorgeCaceres.Facade.JsonBody;
+import JorgeCaceres.entities.JsonProjectBody;
+import org.junit.jupiter.api.Test;
+import org.testng.Assert;
+
+class JsonBodyTest {
```

Comment:
this class name is so generic it looks like is going to test all json bodies of all endpoints